### ISSUE TITLE
Forecasting hedgedocs

### ISSUE THREAD

**Issue by MatKbauer** [ecmwf/WeatherGenerator#2325]
### Overarching
[Weekly planning](https://gitlab.jsc.fz-juelich.de/hedgedoc/UXoFukoqQKqfBLaGPVy-hA?view#Meetings)

### Medium-range
- [O48 ablations](https://gitlab.jsc.fz-juelich.de/hedgedoc/1miF30U4QYOtMnFCF5fdkg#)
- [Ankit's forecast experiments and latent space analyses](https://gitlab.jsc.fz-juelich.de/hedgedoc/HKcbX1EMReag8iGSKJ16bw#)
- [Ankit's dispersive loss explorations](https://gitlab.jsc.fz-juelich.de/hedgedoc/9AWtgbJWSgulvEaybuGfeg#)
- [Early LayerNorm positioning experiments and latitude/channel weighting](https://gitlab.jsc.fz-juelich.de/hedgedoc/i7PlOLUBQX6vge2OAy35Yw#)
- [LayerNorm around skip connection](https://gitlab.jsc.fz-juelich.de/hedgedoc/_dS2ngyWTqmKQHE8BXytDA#)
- [18-step forecast training](https://gitlab.jsc.fz-juelich.de/hedgedoc/AsdQ_MHSRniou_BmAeVmEw#)
- [Constraining latent space for rollout stability](https://gitlab.jsc.fz-juelich.de/hedgedoc/JJvFOcLjSmWwkbUan_BRiw?view#)
- [Savvas' LayerNorm ablations](https://gitlab.jsc.fz-juelich.de/hedgedoc/cUeEqVrBSdSTHYeGf2g77Q#)
- [LayerNorm type experiments](https://gitlab.jsc.fz-juelich.de/hedgedoc/j9c1o19HROSpb-cA7haeVQ#)
- [Effect of LayerNorm for CAMS data](https://gitlab.jsc.fz-juelich.de/hedgedoc/ReWfj7JDRFW-1loP6s2xRA#)
- [Exploring type of latent noise imputed to the forecast engine](https://gitlab.jsc.fz-juelich.de/hedgedoc/Jc6KHms6ToSTaDHHC7crtQ#)
- [Aux info ablations](https://gitlab.jsc.fz-juelich.de/hedgedoc/D9oD31gPTGWRdGdZVR5L7Q#)
- [Role of geoinfo during forecasting](https://gitlab.jsc.fz-juelich.de/hedgedoc/qx5BLjfXTryJRFO7IVz8PQ#)
- [Rollout stability over training progress](https://gitlab.jsc.fz-juelich.de/hedgedoc/dfEZKkWdTOmX-wj16Iq0DA#)
- [Parameter search and first release model details](https://gitlab.jsc.fz-juelich.de/hedgedoc/xKgF-6DDSXyMdzDUFTmgkg?view#)
- [First release model](https://gitlab.jsc.fz-juelich.de/hedgedoc/hJfB69mPQzqFCmoYjYgoXg#)
- [Push-forward exploration Savvas](https://gitlab.jsc.fz-juelich.de/hedgedoc/bGdOXZgDTTyX074RRvliWg#) and [Matze](https://gitlab.jsc.fz-juelich.de/hedgedoc/JJvFOcLjSmWwkbUan_BRiw?view#7-Push-Forward-Trick)
- [Encoder/decoder dimension ablations](https://gitlab.jsc.fz-juelich.de/hedgedoc/QuMEPhb1RZOCySAmY6JvVw#)
- [Ommiting q50 and q100](https://gitlab.jsc.fz-juelich.de/hedgedoc/S1xaC2ZeRkG8ZafAp-pDig#)
- [Target point sampling ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/Me9J-IosT1uddafZ_eyDtQ#)
- [Forecast model candidate 260502 with SwiGLU and XSA](https://gitlab.jsc.fz-juelich.de/hedgedoc/S1YbGks6TM-3RnttY6zzzg#)
- [Christian's 1h models](https://gitlab.jsc.fz-juelich.de/hedgedoc/1UJpbEk3SO2ho6RwxgcdUg#)
- [Time-delta ablations](https://gitlab.jsc.fz-juelich.de/hedgedoc/hmSN3ZMZT7Oil6tW0g_19g?view#)
- [Temperature channel weighting -- not reproducible](https://gitlab.jsc.fz-juelich.de/hedgedoc/e7bAgPVJRg6OvhBiC8xUDA#)
- [Channel weighting part I: AIFS, GraphCast, None, Custom](https://gitlab.jsc.fz-juelich.de/hedgedoc/r9VusP-4QauBIirUK9JUmw#)
- [Channel weighting part II](https://gitlab.jsc.fz-juelich.de/hedgedoc/2Gq8kjRRSTiCSP4YCL6pQQ#)
- [#pre-train and #fine-tune steps](https://gitlab.jsc.fz-juelich.de/hedgedoc/iuP5TDo1Tde2CFuE5A_bkw?view#)

#### ERA5 N320 studies

[ERA5 resolution experiments: O96 vs. N320](https://gitlab.jsc.fz-juelich.de/hedgedoc/S2R1MbagT2STEfFOQ_fn7g?both)
[JEPA with ERA5 Healpix level 5 and 6 studies](https://gitlab.jsc.fz-juelich.de/hedgedoc/s/u2wBtBpfV)
[JEPA n320 experiments](https://gitlab.jsc.fz-juelich.de/hedgedoc/EEkhuYPxQcyi5Aw61gP4Wg)

#### Ablation studies

[Ablation Study - Imputing noise - Summary](https://gitlab.jsc.fz-juelich.de/hedgedoc/Jc6KHms6ToSTaDHHC7crtQ)
[Ablation Study - Imputing Gaussian Latent Noise](https://gitlab.jsc.fz-juelich.de/hedgedoc/6YjxSiyWTf6vvov6aOYGXg)
[Ablation Study - Imputing Laplace noise](https://gitlab.jsc.fz-juelich.de/hedgedoc/l_PXK64ORcaZHxXadzzczg)
[Ablation Study - Imputing Uniform noise](https://gitlab.jsc.fz-juelich.de/hedgedoc/d8rRrhSURpqYwjyRxr8gfA)

### Extended range
- [Ankit's 1000 step rollout with conditionings](https://gitlab.jsc.fz-juelich.de/hedgedoc/1mG4CignQI6G7wMc6bcetg#)
- [First thoughts on climate forcing](https://gitlab.jsc.fz-juelich.de/hedgedoc/-EEOZUm5R0ibEvLIxqV2iQ#)
- [Ankit's latent analysis on extended range stability](https://gitlab.jsc.fz-juelich.de/hedgedoc/-HvFMJutRe2A7qjY4mVJ9Q#)
- [24h vs 24h-average models](https://gitlab.jsc.fz-juelich.de/hedgedoc/hmSN3ZMZT7Oil6tW0g_19g?view#6x1h-vs-6h-vs-24h-vs-24h-averaged-80-day-rollout)

Please add other HedgeDocs or sources directly to this post or add them to the comment section.

### Area

- [ ] datasets, data readers, data preparation and transfer
- [x] model
- [ ] science
- [ ] infrastructure and engineering
- [ ] evaluation, export and visualization
- [ ] documentation
- [ ] performance
